modules/distr-tracing-tempo-config-storage.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-storage_{context}"]
+= Distributed tracing storage configuration options
+
+You configure storage for the Collector, Ingester, Distributor, and Query services under `spec.storage`. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
```

Comment:
```suggestion

You can configure tempo storage options under `spec.storage`. Different storage options could be provided depending on the object storage backend used.

General storage parameters used by the {TempoOperator} to define distributed tracing storage
[options="header"]
[cols="l, a, a, a"]
|===
|Parameter |Description |Values |Default value
spec:
  storage:
    secret
       type:
|Type of storage to use for the deployment., could be azure, s3, gcs
|storage:
  secretname:
|Name of the secret, that contains the object storage configuration for example `tracing-secret`. each secret will contains storate configuration depending on the type
|storage:
  tls:
   caName:
|CA is the name of a ConfigMap containing a CA certificate.

Reply:
I've transferred and edited this suggestion manually. Resolving this conversation.